cli.js

```diff
@@ -0,0 +1,36 @@
+let colors = require("colors");
+
+const { readFile, readFileValidation, readFilesStats } = require("./index.js");
+
+const options = {
+  path: {
+    demand: true,
+    alias: "p",
+    default: "README.md",
+  },
+};
+
+const argv = require("yargs")
+  .command("validate", " result of links validation in a Markdow file", options)
+  .command("stats", " stats of all", options)
+  .command("validateStats", "Flag indicating if links are validated", options)
```

Comment:
el usuario puede perderse con el uso de camela case.
```suggestion
  .command("validate-stats", "Flag indicating if links are validated", options)
```